routes/web.php

```diff
@@ -29,6 +29,6 @@
 
 Route::resource('orders', 'OrderController');
 
-Auth::routes();
+Route::post('/daily-menus/editMenuItem', 'DailyMenuController@updateMenuItem'); //route for ajax request
```

Comment:
I was planning that i use this route for update quantity of food in a menu of date, and route::resource for update will update all of item in a menu